elastic/beats
# metricbeat: Update beat module with apm-server monitoring metrics fields

## Proposed commit message



- Update metricbeat beat module mapping to contain latest apm-server monitoring metrics fields
- Update schema to parse `output` fields from stats
- Update tests

## Checklist




## Disruptive User Impact



## Author's Checklist



## How to test this PR locally


- Start apm-server with monitoring
- Start standalone metricbeat with beat module
- Confirm that fields are mapped correctly in `metricbeat-*` data stream

Manual testing results described in https://github.com/elastic/apm-server/issues/13475

## Related issues


- Part of https://github.com/elastic/apm-server/issues/13475

## Use cases



## Screenshots



## Logs
